Assets/Scripts/Controllers/AI/Boss/Boss.cs

```diff
@@ -37,7 +36,7 @@ public class Boss : MonoBehaviour, ISaveable {
 			if (obj == gameObject) {
 				_isAlive = false;
 				gameObject.SetActive(false);
-				SceneManager.LoadScene(SceneDefs.EndingScreen);
+				SceneManager.LoadScene(SceneDefs.OutroCutscene);
 			}
 		});
```

Comment:
1. Consider putting this into a new method that isn't anonymous/lambda.
2. Could use some sort of fade-out animation (for the expo), but this works for now.

Reply:
Why does it need a new methode i just changed the scene name. when we didn't had the cutscene it was fine but now it needs a new method. Why?

Reply:
Just a general code review. To me, it would be just a bit more nice to read (less cluttered).